### PR TITLE
Update Deploying to ZEIT Now step in /learn

### LINKED ISSUE #194: Improvements needed for deployment step in /learn
From a user:
> I was going through the Next js tutorial and came to this point: https://nextjs.org/learn/basics/deploying-a-nextjs-app/deploying-to-zeit-now
>
> When I ran now at first, the build failed, because there was no now-build script and no next.config.js. Googling the build error send me here: https://github.com/zeit/now-builders/blob/master/errors/now-next-no-serverless-pages-built.md, which helped me solve it, but I guess it would be helpful to add this to the tutorial.

We should reference both `next.config.js` as the entry point with the new @now/next, include a `now-build` script in `package.json`, and link to the Now guide for more information on things like caching and more resources.

### PR DESCRIPTION
**Pull request by kheruc**
Fix #194

### CODE REVIEW

File: learn/pages/learn/basics/deploying-a-nextjs-app/deploying-to-zeit-now.mdx [hunk 1]
```diff
@@ -35,10 +34,12 @@ Then, create a now.json file in the root of your project with the following cont
 ```json
 {
   "version": 2,
-  "builds": [{ "src": "package.json", "use": "@now/next" }]
+  "builds": [{ "src": "next.config.js", "use": "@now/next" }]
```
Comment (timneutkens) [inline learn/pages/learn/basics/deploying-a-nextjs-app/deploying-to-zeit-now.mdx]: This is actually correct

```suggestion
  "builds": [{ "src": "package.json", "use": "@now/next" }]
```

Comment (kheruc) [reply learn/pages/learn/basics/deploying-a-nextjs-app/deploying-to-zeit-now.mdx]: I was following https://zeit.co/docs/v2/deployments/official-builders/next-js-now-next/
But either works

Comment (kheruc) [reply learn/pages/learn/basics/deploying-a-nextjs-app/deploying-to-zeit-now.mdx]: The official guide also uses "next.config.js" https://zeit.co/guides/deploying-nextjs-with-now/

Comment (timneutkens) [reply learn/pages/learn/basics/deploying-a-nextjs-app/deploying-to-zeit-now.mdx]: Let's update that to use package.json 👍

File: learn/pages/learn/basics/deploying-a-nextjs-app/deploying-to-zeit-now.mdx [hunk 2]
```diff
@@ -39,6 +47,8 @@ Then, create a now.json file in the root of your project with the following cont
 }
 ```
 
+> The [@now/next Builder](https://zeit.co/docs/v2/deployments/official-builders/next-js-now-next) will convert pages of our app into a series of individual [lambdas](https://zeit.co/docs/v2/deployments/concepts/lambdas/). Read a [Create a Next.js Application and Deploy with Now](https://zeit.co/guides/deploying-nextjs-with-now/) guide for additional information.
```
Comment (timneutkens) [inline learn/pages/learn/basics/deploying-a-nextjs-app/deploying-to-zeit-now.mdx]:
```suggestion
> The [@now/next Builder](https://zeit.co/docs/v2/deployments/official-builders/next-js-now-next) will convert pages of our app into a series of individual [lambdas](https://zeit.co/docs/v2/deployments/concepts/lambdas/). Read the [Create a Next.js Application and Deploy with Now](https://zeit.co/guides/deploying-nextjs-with-now/) guide for additional information.
```

**Pull request merged**